test/scripts/box/file.js

```diff
@@ -24,31 +23,37 @@ describe('File', () => {
     '- Banana'
   ].join('\n');
 
-  const obj = yaml.load(body);
+  const obj = load(body);
   const path = 'test.yml';
 
-  function makeFile(path, props) {
+  const makeFile = (path, props) => {
     return new File(Object.assign({
-      source: pathFn.join(box.base, path),
+      source: join(box.base, path),
       path
     }, props));
-  }
+  };
 
   const file = makeFile(path, {
-    source: pathFn.join(box.base, path),
+    source: join(box.base, path),
     path,
     type: 'create',
     params: {foo: 'bar'}
   });
 
-  before(() => Promise.all([
-    fs.writeFile(file.source, body),
-    hexo.init()
-  ]).then(() => fs.stat(file.source)));
+  before(async() => {
```

Comment:
Should we add space between `async` and `()`? (e.g. async () => {}).

Related eslint rule: [`space-before-function-paren`](https://eslint.org/docs/rules/space-before-function-paren).

Reply:
Updated syntax